src/plugins/ui/evaluation.ts

```diff
@@ -190,10 +190,9 @@ export class EvaluationPlugin extends UIPlugin {
     const getters = this.getters;
     function readCell(range: Range): any {
       let cell: Cell | undefined;
-      if (!getters.tryGetSheet(range.sheetId)) {
-        throw new Error(_lt("Invalid sheet name"));
+      if (getters.getSheet(range.sheetId)) {
+        cell = getters.getCell(range.sheetId, range.zone.left, range.zone.top);
```

Comment:
Why this change ?

Reply:
Because we used the getter `getSheet` in the fix for 15.0 and saas-15.1. It checks for the same condition and throws the error as well. So I made this change for both consistency and optimization. However, I can undo it if you want me to : )
@pro-odoo

Reply:
Yes, you can undo this change. It's easier to know why we try to get the sheet.

Reply:
Done! @pro-odoo